_opensearch/ux.md

```diff
@@ -39,7 +39,7 @@ These methods are described in the following sections.
 Prefix matching finds documents that matches the last term in the query string.
 
 For example, assume that the user types “qui” into a search UI. To autocomplete this phrase, use the `match_phrase_prefix` query to search all `text_entry` fields that begin with the prefix "qui."
-To make the word order and relative positions flexible, specify a `slop` value. To learn about the `slop` option, see [Other optional query fields]({{site.url}}{{site.baseurl}}/opensearch/query-dsl/full-text#other-optional-query-fields).
+To make the word order and relative positions flexible, specify a `slop` value. To learn about the `slop` option, see the [Multi-match query]({{site.url}}{{site.baseurl}}/opensearch/query-dsl/full-text#multi-match).
```

Comment:
"Query" isn't in the name of the linked heading - should be removed from the brackets (add it after the link if it's needed for context).

```
To make the word order and relative positions flexible, specify a `slop` value. To learn about the `slop` option, see the [Multi-match]({{site.url}}{{site.baseurl}}/opensearch/query-dsl/full-text#multi-match).
```